pkg/webhook/packagebundlecontroller_webhook.go

```diff
@@ -92,15 +101,15 @@ func (v *activeBundleValidator) handleInner(_ context.Context, pbc *v1alpha1.Pac
 	}
 
 	if !found {
-		msg := fmt.Sprintf("activeBundle <%q> not present on cluster", pbc.Spec.ActiveBundle)
+		reason := fmt.Sprintf("activeBundle %q not present on cluster", pbc.Spec.ActiveBundle)
```

Comment:
```suggestion
		reason := fmt.Sprintf("active bundle %q is not present on cluster", pbc.Spec.ActiveBundle)
```
I like dropping the <>, and since this is for a human, I'd take it this little step further.

Would it make sense/be simple to add:
```suggestion
		reason := fmt.Sprintf("active bundle %q is not present on cluster %q", pbc.Spec.ActiveBundle, clusterName)
```
?? I don't know if the clusterName is handy in the environment.